CHANGELOG.md

```diff
@@ -0,0 +1,58 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
```

Comment:
I would like to consider using a file per version, like for Kubo, aside of rendering issues (which should be fine for a while). I just like having a per version file because this is easy to link to and always have the ToC at the right place even if newer versions are released.

Feel free to resolve this if you want to keep this like this.

Reply:
The intention here is to use https://github.com/pl-strflt/changelog-driven-release so that the changelogs drive the release version and the GitHub release notes. The benefit is that there isn't a scramble at release time to assemble the changelogs, they get accumulated in [Unreleased] and then you change [Unrelease] to [vX.Y.Z] and the release happens. The goal is to reduce the cost of cutting a release so that we cut them more often.

Reply:
I think @Jorropo has a good point about the durability of the links when we do one file per version.  Related, it makes the URLs easily hackable.  For example I know I can always do https://github.com/ipfs/kubo/blob/master/docs/changelogs/v0.Y.md#-highlights to get the highlights section.

I don't think this is a blocker, but maybe a feature request to add to https://github.com/pl-strflt/changelog-driven-release about supporting multiple files.